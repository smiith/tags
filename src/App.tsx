import React from 'react';
import Tags from './Tags';
import Container from 'react-bootstrap/Container';

function App() {
	return (
		<Container className="p-3">
			<Container className="p-5 mb-4 bg-light rounded-3">
				<h1 className="header">Tags demo</h1>
				<Tags />
				<Tags />
				<Tags />
			</Container>
		</Container>
	);
}

export default App;
