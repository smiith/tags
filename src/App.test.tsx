import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('App view render snapshot', () => {
	const view = render(<App />);
	expect(view).toMatchSnapshot();
});
