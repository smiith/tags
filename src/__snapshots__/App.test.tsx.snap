// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App view render snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="p-3 container"
      >
        <div
          class="p-5 mb-4 bg-light rounded-3 container"
        >
          <h1
            class="header"
          >
            Tags demo
          </h1>
          <div>
            <div
              class="row"
            >
              <div
                class="col"
              >
                <div
                  class="input-group"
                >
                  <input
                    class="form-control"
                    placeholder="Tag"
                    value=""
                  />
                </div>
              </div>
              <div
                class="col"
              >
                <button
                  class="btn btn-primary"
                  type="button"
                >
                  Add Tag
                </button>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="p-3 col-6"
              >
                <div
                  class="list-group"
                />
              </div>
            </div>
          </div>
          <div>
            <div
              class="row"
            >
              <div
                class="col"
              >
                <div
                  class="input-group"
                >
                  <input
                    class="form-control"
                    placeholder="Tag"
                    value=""
                  />
                </div>
              </div>
              <div
                class="col"
              >
                <button
                  class="btn btn-primary"
                  type="button"
                >
                  Add Tag
                </button>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="p-3 col-6"
              >
                <div
                  class="list-group"
                />
              </div>
            </div>
          </div>
          <div>
            <div
              class="row"
            >
              <div
                class="col"
              >
                <div
                  class="input-group"
                >
                  <input
                    class="form-control"
                    placeholder="Tag"
                    value=""
                  />
                </div>
              </div>
              <div
                class="col"
              >
                <button
                  class="btn btn-primary"
                  type="button"
                >
                  Add Tag
                </button>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="p-3 col-6"
              >
                <div
                  class="list-group"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="p-3 container"
    >
      <div
        class="p-5 mb-4 bg-light rounded-3 container"
      >
        <h1
          class="header"
        >
          Tags demo
        </h1>
        <div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="input-group"
              >
                <input
                  class="form-control"
                  placeholder="Tag"
                  value=""
                />
              </div>
            </div>
            <div
              class="col"
            >
              <button
                class="btn btn-primary"
                type="button"
              >
                Add Tag
              </button>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="p-3 col-6"
            >
              <div
                class="list-group"
              />
            </div>
          </div>
        </div>
        <div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="input-group"
              >
                <input
                  class="form-control"
                  placeholder="Tag"
                  value=""
                />
              </div>
            </div>
            <div
              class="col"
            >
              <button
                class="btn btn-primary"
                type="button"
              >
                Add Tag
              </button>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="p-3 col-6"
            >
              <div
                class="list-group"
              />
            </div>
          </div>
        </div>
        <div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="input-group"
              >
                <input
                  class="form-control"
                  placeholder="Tag"
                  value=""
                />
              </div>
            </div>
            <div
              class="col"
            >
              <button
                class="btn btn-primary"
                type="button"
              >
                Add Tag
              </button>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="p-3 col-6"
            >
              <div
                class="list-group"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
